src/components/cards.css

```diff
@@ -0,0 +1,146 @@
+.cards {
+  background: transparent;
+  padding: 4rem;
+}
+
+h1 {
+  text-align: center;
+}
+
+h3 {
+  color: rgb(255, 60, 255);
+  font-family: "Audiowide";
+  font-size: 30px;
+  text-align: center;
+}
+
+#card-index0 {
+  left: -700px;
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1;
+}
+
+#card-index1 {
+  left: 700px;
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1;
+}
```

Comment:
Same applies here with overlapping css but only minor changes, the comment about the section-id's